source/game/field/obj/ObjectWoodbox.hh

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "game/field/obj/ObjectBreakable.hh"
+
+namespace Field {
+
+class ObjectWoodbox : public ObjectBreakable {
```

Comment:
Does anything inherit from this class? I think no, so I would prefer if we started labeling as final
```suggestion
class ObjectWoodbox final : public ObjectBreakable {
```